pkg/consistent/consistent.go

```
@@ -0,0 +1,492 @@
+	if config.Load == 0 {
+		config.Load = DefaultLoad
+	}
+	if config.PickerWidth == 0 {
+		config.PickerWidth = DefaultPickerWidth
+	}
+
+	// Initialize a new Consistent instance with the provided configuration.
+	c := &Consistent{
+		config:         config,
+		members:        make(map[string]*Member),
+		partitionCount: uint64(config.PartitionCount),
```

golangci-lint (1.62.2)

[high] G115: integer overflow conversion int -> uint64

(gosec)